src/components/color_picker/color_palette_picker/color_palette_picker.tsx

```diff
@@ -62,9 +62,10 @@ export interface EuiColorPalettePickerPaletteFixedProps extends CommonProps {
    */
   type: 'fixed';
   /**
-   * Array of color `strings`.
+   * Array of color `strings` or `ColorStops` in the form of
```

Comment:
Everywhere we mention `ColorStops`, can you also import this type and display it in the props tab using the new props loader and then add the link style props comment?
```suggestion
   * Array of color `strings` or #ColorStop in the form of
```

<img width="986" alt="Screen Shot 2020-10-30 at 10 58 23 AM" src="https://user-images.githubusercontent.com/549577/97720872-29e61980-1a9f-11eb-9b35-b889f9898434.png">